pkg/kubelet/preemption/preemption.go

```diff
@@ -205,6 +206,9 @@ func (a admissionRequirementList) subtract(pods ...*v1.Pod) admissionRequirement
 		newQuantity := req.quantity
 		for _, pod := range pods {
 			newQuantity -= resource.GetResourceRequest(pod, req.resourceName)
+			if newQuantity <= 0 {
+				break
+			}
 		}
 		if newQuantity > 0 {
```

Comment:
I believe we can get rid of this `newQuantity > 0` check right? Because we would've already broken out of the loop if this was true?

Reply:
If newQuantity > 0 is true, the break on line 210 would not run.
We still need to check this condition here.

Reply:
Mhhm, yes you are 100% right :)